pages/tasks/task_details.py

```diff
@@ -11,6 +11,7 @@
 class TaskDetailsPage(Base):
 
     _name_locator = (By.CSS_SELECTOR, '.content-container > h1')
+    _team_locator = (By.CSS_SELECTOR, '.three-fields > a')
```

Comment:
Your locator syntax was close. :)  I believe what you need is:
`(By.CSS_SELECTOR, 'a[href*="team"]')`

There's a really good css/xpath locator cheat sheet available at https://www.simple-talk.com/dotnet/.net-framework/xpath,-css,-dom-and-selenium-the-rosetta-stone/ which I refer to all the time.